packages/graphql/src/services/liveQuizzes.ts

```diff
@@ -1553,7 +1553,7 @@ function aggregateLiveQuizResponses({
                         criterionId
                       ] === 'undefined'
                     ) {
-                      return
+                      return acc
```

Comment:
by returning early we do not add invalid data to the results -> student submissions might be processed partially, however